utils/docker-compose/README.md

```diff
@@ -0,0 +1,225 @@
+- [orderer.yaml](network-config/orderer.yaml) : Orderer configuration
+- [crypto-config.yaml](network-config/crypto-config.yaml) : File used with cryptogen to generate
+certs for specified number of orgs, peers, users, and orderer. The CA credentials
+can be used to start instances of fabric-ca
+- [configtx.yaml](network-config/configtx.yaml)  : File used with configtxgen to generate the
+genesis block which is used as the basis of the specified channel
+- [docker-compose.yml](network-config/docker-compose.yml) : Configuration of the
+fabric network to be used with `docker-compose`. This file depends on two
+environment variables to properly bring up a network. `$FPC_CFG` can be set to
+`-fpc` or shall be empty. If set to `-fpc` the `core-fpc.yaml` & FPC peer image
+is used. Otherwise it will use `core.yaml` and the regular peer image.
+`$PEER_CMD` must also be set to the location of binary or script that will start
+ the peer.  **Docker version 17.06.2-ce or higher is needed**
+
+## Steps
+1. Download the necessary fabric binaries. Run the
+[bootstrap script](scripts/bootstrap.sh) which will download the Fabric 1.4.3
+into a local bin directory. If you already have the binaries downloaded in your
+path, this step can be skipped. **Fabric 1.4.3 versions of configtxgen and
+cryptogen are required to use the configurations above.**
+```
+./scripts/bootstrap.sh
+```
+1. Build the peer image in `utils/docker/peer` directory which is defined by the
+peer [Dockerfile](../docker/peer/Dockerfile). This step
+assumes you have already built the [fabric-private-chaincode base image](../docker/base/Dockerfile).
+Take a look at building the docker dev environment in the main [README](../../README.md#docker).
+After you have created the base image, run the following to create a modified
+peer image and the plugins necessary to start the peer.  `$FPC_PATH` is the
+location fabric-private-chaincode repository on your host machine.
+```
+cd $FPC_PATH/utils/docker/peer
+docker build -t hyperledger/fabric-peer-fpc .
+```
+By default the image will clone the master branch on
+https://github.com/hyperledger-labs/fabric-private-chaincode. If you want to use
+a different fork of the repo or a different branch you provide
+FPC_REPO_URL and FPC_REPO_BRANCH as build args.
+```
+cd $FPC_PATH/utils/docker/peer
+docker build -t hyperledger/fabric-peer-fpc --build-arg FPC_REPO_URL=<repo-url> --build-arg FPC_REPO_BRANCH=<repo-branch> .
+```
+If you want to build the peer image using your local copy of your repo you can
+use the same build args, but specify `file:///tmp/build-src/.git` as the
+`FPC_REPO_URL`. You will also need to create the image at the root of this repo
+so that the local repo will be in the build context for the docker daemon.
+```
+cd $FPC_PATH
+docker build -t hyperledger/fabric-peer-fpc -f utils/docker/peer/Dockerfile --build-arg FPC_REPO_URL=file:///<your-gopath>/src/github.com/hyperledger-labs/fabric-private-chaincode.git .
```

Comment:
this should be "....../fabric-private-chaincode.git/.git ."?  And given that you already assume $FPC_PATH, maybe even simpler write it as "...-build-arg FPC_REPO_URL=file:///${FPC_PATH}/.git"  (or to be more robust "...-build-arg FPC_REPO_URL=file:///$(readlink -f {FPC_PATH})/.git"?